src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -307,6 +351,7 @@ class HttpGoogleCloudStorageDAO(
       .build()
   }
 
+  def adminGroupName = s"${groupsPrefix}-ADMIN@${appsDomain}"
```

Comment:
I think this needs to be more configurable unless we are careful that each environment has a different value for appsDomain